### ISSUE TITLE
executable file not found in $PATH

### ISSUE THREAD

**Issue by lettucetrail** [itzg/docker-minecraft-server#762]
Pulled latest version and will not start

[init] Fixing PATH to include java
[init] Starting the Minecraft server...
2021-02-11T02:54:46.255Z	ERROR	mc-server-runner	Failed to start	{"error": "exec: \"java\": executable file not found in $PATH"}
2021-02-11T02:54:46.255Z	ERROR	mc-server-runner	command failed abnormally	{"error": "exec: not started"}
2021-02-11T02:54:46.256Z	INFO	mc-server-runner	Done

Containers built on older images are fine, newly created containers experience the same issue

**Comment by james-d-elliott** [comment]
What's your docker env config look like for the container in question?

**Comment by lettucetrail** [comment]
JAVA_VERSION jdk-11.0.10+9
JAVA_HOME /opt/java/openjdk 
JVM_XX_OPTS -XX:+UseG1GC

These seem new and unfamiliar to me

**Comment by lettucetrail** [comment]
Changed image to multiarch-latest, container started, went back to multiarch and all is well.

**Comment by itzg** [comment]
@lettucetrail I'm glad it finally worked out for you. I'm gathering some data on these scenarios, so I was wondering what platform you're using to run your containers? Command line Docker, a NAS like Synology, etc?

**Comment by lettucetrail** [comment]
Sure sure, lemmie give you some excess detail then. 

Command line Docker. It's Docker installed on a server that I mainly access through portainer.
I decided to reboot my server to fix a mod related issue, at this point I was running the multiarch branch. 
When I restarted I decided to change the way the container attaches to the console which caused the image to pull and update.
I have a couple older containers (from previous modpacks/versions) and they were able to boot up just fine.
I was receiving exit code 1 so I removed the /start parameter and was able to load up the container and see that it was receiving those errors related to path
I tried switching the container's image to server:latest but the issue persisted
Back to multiarch, still there
Then I realised there's also multiarch-latest and tried that which fixed it

These are the majority of my ENV config
`EULA 	TRUE
FTB_MODPACK_ID 	79
ALLOW_FLIGHT 	TRUE
SERVER_NAME 	Carl
DIFFICULTY 	hard
ALLOW_NETHER 	true
SNOOPER_ENABLED 	false
SPAWN_PROTECTION 	0
MOTD 	GLHFDD
PATH 	/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
UID 	1000
GID 	1000
MEMORY 	8G
VERSION 	LATEST
FORGEVERSION 	RECOMMENDED
SPONGEBRANCH 	STABLE
FABRICVERSION 	LATEST
PVP 	true
ENABLE_RCON 	true
ONLINE_MODE 	TRUE
ENABLE_AUTOPAUSE 	false
AUTOPAUSE_TIMEOUT_EST 	3600
AUTOPAUSE_TIMEOUT_KN 	120
AUTOPAUSE_TIMEOUT_INIT 	600
AUTOPAUSE_PERIOD 	10
DISABLE_HEALTHCHECK 	true
AUTOPAUSE_KNOCK_INTERFACE 	eth0
LEVEL_TYPE 	DEFAULT
LANG 	en_US.UTF-8
LANGUAGE 	en_US:en
LC_ALL 	en_US.UTF-8
JAVA_VERSION 	jdk-11.0.10+9
JAVA_HOME 	/opt/java/openjdk
JVM_XX_OPTS 	-XX:+UseG1GC`

The things related to lang and below look new to me, I think they got added when it pulled the new image?
